### DISCUSSION TITLE
Survival Tree Sample Weights

### DISCUSSION THREAD

**Discussion by ThaliaSeale** [sebp/scikit-survival#279] [Q&A]
I am using sample weights when fitting a survival tree/random survival forest. I am trying to understand how the sample weights affect the log-rank criterion. In standard decision trees, this involves each sample's contribution to the impurity measure, however, I'm not sure exactly how this would work with the log-rank criterion. Any insight would be appreciated.

**Comment by sebp** [comment]
It doesn't affect the log-rank statistic itself. If you set `min_weight_fraction_leaf` to something other than 0, the sample weights will be used to decide when to stop growing the tree.
